<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPIPQTClass</class>
 <widget class="QMainWindow" name="TCPIPQTClass">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat Client</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_username">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Username :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Password :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Entrez un nom d'utilisateur</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordEdit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Entrez un mot de passe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSignUptButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>S'inscrire</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMessageButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Status : Aucun</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushConnectButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>260</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Se connecter</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="MessageEdit">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>641</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="placeholderText">
     <string>Entrez le texte souhaiter ....</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_error">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_message">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>701</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>768</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="TCPIPQT.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushConnectButton</sender>
   <signal>clicked()</signal>
   <receiver>TCPIPQTClass</receiver>
   <slot>onConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushMessageButton</sender>
   <signal>clicked()</signal>
   <receiver>TCPIPQTClass</receiver>
   <slot>onSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushSignUptButton</sender>
   <signal>clicked()</signal>
   <receiver>TCPIPQTClass</receiver>
   <slot>onSignUpButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>275</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConnectButtonClicked()</slot>
  <slot>onSendMessageButtonClicked()</slot>
  <slot>onSignUpButtonClicked()</slot>
 </slots>
</ui>
